interop/jvm/src/main/scala/scalaz/zio/interop/catz.scala

```diff
@@ -16,8 +16,9 @@ object catz extends RTS {
       val cbZ2C: ExitResult[Throwable, A] => Either[Throwable, A] = {
         case ExitResult.Completed(a)       => Right(a)
         case ExitResult.Failed(t, _)       => Left(t)
-        case ExitResult.Terminated(Nil)    => Left(Errors.TerminatedFiber)
-        case ExitResult.Terminated(t :: _) => Left(t)
+        case ExitResult.Terminated(Nil, Nil)    => Left(Errors.TerminatedFiber)
+        case ExitResult.Terminated(Nil, t :: _) => Left(t)
```

Comment:
Don't think it makes sense to add this here instead of `TerminatedFiber` since it's not the cause of termination.